data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json

```diff
@@ -1,211 +1,380 @@
+                "Zoology",
+                "Ecology",
+                "Other Biodiversity",
+                "Other Geodiversity",
+                "Unclassified"
+            ],
+            "examples": [
+                "Botany"
+            ]
+        },
+        "schema:taxonomicRange": {
+            "type": "array",
+            "description": "The taxonomic grouping of the organism.",
+            "items": {
+                "type": "string"
+            }
+        },
+        "schema:documentation": {
+            "type": "string",
+            "description": "URL to further documentation describing the service in more detail.",
+            "examples": [
+                "cetaf.org"
+            ]
+        },
+        "schema:ratingValue": {
+            "type": "number",
+            "description": "The rating for the content",
+            "examples": [
+                0,
+                50,
+                100
             ]
-          ]
+        },
+        "schema:ratingExplanation": {
+            "type": "string",
+            "description": "A short explanation (e.g. one to two sentences) providing background context and other information that led to the conclusion expressed in the rating.",
+            "examples": [
+                "This service is accepted, because...",
+                "This service is rejected, because..."
+            ]
+        },
+        "schema:Service": {
+            "type": "object",
+            "description": "A service provided by an organization, e.g. delivery service, print services, etc.",
+            "properties": {
+                "schema:serviceType": {
+                    "type": "string",
+                    "description": "A type that defines the kind of taxonomic service",
+                    "enum": [
+                        "AITrainingDataset",
+                        "communityGroup",
+                        "crowdSourcing",
+                        "dataTool",
+                        "e-LearningService",
+                        "factsheet",
+                        "identification",
+                        "inventory",
+                        "referenceCollection",
+                        "specimenDatasetNotInGBIF",
+                        "website"
+                    ]
+                },
+                "schema:name": {
+                    "type": "string",
+                    "description": "The name of the Servie",
+                    "examples": [
+                        "Bombus reference collection"
+                    ]
+                },
+                "schema:description": {
+                    "type": "string",
+                    "description": "A description of the Service",
+                    "examples": [
+                        "A Bombus reference collection that contains a bunch of different specimen related to the Bombus species"
+                    ]
+                },
+                "schema:slogan": {
```

Comment:
Why is this here? It seems unnecessary

Reply:
It was in the original schema of the EOSC Marketplace we based the schema on. That is why it is still in there, but it was first called 'tagLine'. We use it in the marketplace as a kind of subtitle, its up to @wouteraddink if the thinks this field is necessary or not. I think it does add some value, but not that much that we could not live without it.